### DISCUSSION TITLE
What's the best way to discuss a possible security issue with scala-xml?

### DISCUSSION THREAD

**Discussion by beny23** [scala/scala-xml#674] [Q&A] [answered]
As per title - would I just raise an issue or is there a way to discuss privately?

**Comment by dubinsky** [comment]
@SethTisue please weigh in ^^

**Comment by lrytz** [comment, marked as answer]
@beny23 thank you for asking! If you don't mind, could you please send the issue to security@lightbend.com?

This made us realize we need a security page on scala-lang.org, I think https://akka.io/security/ is a good example. We will follow up on that.

**Reply by beny23** [reply]
Thank you, email sent.
